src/components/FormSubmit/index.native.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {View} from 'react-native';
+import {FormSubmitProps, FormSubmitRef} from './types';
+
+function FormSubmit({style, children}: FormSubmitProps, ref: FormSubmitRef) {
+    return (
+        <View
+            ref={ref}
+            style={style}
+        >
+            {children}
+        </View>
+    );
+}
+
+FormSubmit.displayName = 'FormSubmit';
+
+const FormSubmitWithRef = React.forwardRef(FormSubmit);
+
+export default FormSubmitWithRef;
```

Comment:
can we avoid the extra variable creation? and do
```
export default React.forwardRef
```

Reply:
removed